src/ConfigWidgets/SymbolLocationsDialogTest.cpp

```diff
@@ -97,14 +119,36 @@ class SymbolLocationsDialogTest : public ::testing::Test {
         },
         Qt::QueuedConnection);
   }
+  void SetExpectMetricOpenWithoutModule() {
```

Comment:
Nit: Maybe it's a pattern used before (in this case ignore my comment), but I would erase "Set" in the name of these functions

Reply:
done :)